src/inverseRobot.js

```diff
@@ -24,7 +24,21 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const obj = {};
+  const robotValues = Object.values(robot);
+  const noRepeated = robotValues.filter((item, index) => {
+    return robotValues.indexOf(item) === index;
+  });
+
+  if (robotValues.length !== noRepeated.length) {
+    return null;
+  }
```

Comment:
it's not a boolean, it's still an array

```suggestion
  const uniqueValues = robotValues.filter((item, index) => {
    return robotValues.indexOf(item) === index;
  });

  if (robotValues.length !== uniqueValues.length) {
    return null;
  }
```